tests/unit_tests/test_features.py

```diff
@@ -1,14 +1,26 @@
 import pytest
 
 from tests.utilities import generate_email_list
-from tests.utilities.mocks.mock_variants import VARIANTS
+from tests.utilities.mocks.mock_variants import VARIANTS, VARIANTS_WITH_STICKINESS
 from tests.utilities.testing_constants import IP_LIST
 from UnleashClient.features import Feature
-from UnleashClient.strategies import Default, RemoteAddress, UserWithId
+from UnleashClient.strategies import Default, FlexibleRollout, RemoteAddress, UserWithId
 from UnleashClient.variants import Variants
 
 (EMAIL_LIST, CONTEXT) = generate_email_list(20)
 
+BASE_FLEXIBLE_ROLLOUT_DICT = {
+    "name": "flexibleRollout",
+    "parameters": {"rollout": 50, "stickiness": "userId", "groupId": "AB12A"},
+    "variants": VARIANTS_WITH_STICKINESS,
+    "constraints": [
+        {"contextName": "environment", "operator": "IN", "values": ["staging", "prod"]},
```

Comment:
I think the constraints here are noise, we should be able to do the same test with rollout 0/100 and have it a lot clearer without digging into the weeds of constraints if something goes wrong and needs debugging